ERROR - 2024-07-09 15:28:34 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4\project1-ulfa\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'ci4', 3306, '', 0)
#1 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\BaseConnection.php(1481): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\BaseConnection.php(1500): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\MigrationRunner.php(163): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\ci4\project1-ulfa\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ci4\project1-ulfa\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ci4\project1-ulfa\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ci4\project1-ulfa\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ci4\project1-ulfa\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ci4\project1-ulfa\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\ci4\project1-ulfa\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\BaseConnection.php(1481): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\BaseConnection.php(1500): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\MigrationRunner.php(163): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\ci4\project1-ulfa\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\ci4\project1-ulfa\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\ci4\project1-ulfa\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\ci4\project1-ulfa\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp\htdocs\ci4\project1-ulfa\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#11 C:\xampp\htdocs\ci4\project1-ulfa\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#12 {main}
ERROR - 2024-07-09 15:37:42 --> mysqli_sql_exception: Table 'ci4.tasks' doesn't exist in C:\xampp\htdocs\ci4\project1-ulfa\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\MySQLi\Connection.php(310): mysqli->query('INSERT INTO `ta...', 0)
#1 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ta...')
#2 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ta...')
#3 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ta...', Array, false)
#4 C:\xampp\htdocs\ci4\project1-ulfa\app\Database\Seeds\TaskSeeder.php(11): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\Seeder.php(147): App\Database\Seeds\TaskSeeder->run()
#6 C:\xampp\htdocs\ci4\project1-ulfa\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#7 C:\xampp\htdocs\ci4\project1-ulfa\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run(Array)
#8 C:\xampp\htdocs\ci4\project1-ulfa\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('db:seed', Array)
#9 C:\xampp\htdocs\ci4\project1-ulfa\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp\htdocs\ci4\project1-ulfa\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#11 C:\xampp\htdocs\ci4\project1-ulfa\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#12 {main}
ERROR - 2024-07-09 15:38:58 --> mysqli_sql_exception: Table 'ci4.tasks' doesn't exist in C:\xampp\htdocs\ci4\project1-ulfa\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\MySQLi\Connection.php(310): mysqli->query('INSERT INTO `ta...', 0)
#1 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ta...')
#2 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ta...')
#3 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ta...', Array, false)
#4 C:\xampp\htdocs\ci4\project1-ulfa\app\Database\Seeds\TaskSeeder.php(11): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\Seeder.php(147): App\Database\Seeds\TaskSeeder->run()
#6 C:\xampp\htdocs\ci4\project1-ulfa\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#7 C:\xampp\htdocs\ci4\project1-ulfa\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Seed->run(Array)
#8 C:\xampp\htdocs\ci4\project1-ulfa\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('db:seed', Array)
#9 C:\xampp\htdocs\ci4\project1-ulfa\system\Boot.php(351): CodeIgniter\CLI\Console->run()
#10 C:\xampp\htdocs\ci4\project1-ulfa\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#11 C:\xampp\htdocs\ci4\project1-ulfa\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#12 {main}
ERROR - 2024-07-09 16:40:53 --> mysqli_sql_exception: Table 'ci4.tasks' doesn't exist in C:\xampp\htdocs\ci4\project1-ulfa\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\MySQLi\Connection.php(310): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ci4\project1-ulfa\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4\project1-ulfa\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\ci4\project1-ulfa\app\Controllers\TaskController.php(19): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\ci4\project1-ulfa\system\CodeIgniter.php(933): App\Controllers\TaskController->index()
#8 C:\xampp\htdocs\ci4\project1-ulfa\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TaskController))
#9 C:\xampp\htdocs\ci4\project1-ulfa\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4\project1-ulfa\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4\project1-ulfa\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ci4\project1-ulfa\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\ci4\project1-ulfa\system\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-07-09 16:40:53 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'ci4.tasks' doesn't exist
[Method: GET, Route: task]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tasks`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\TaskController.php(19): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\TaskController->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TaskController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\ci4\\project1-ulfa\\public\\index.php')
CRITICAL - 2024-07-09 16:40:53 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'ci4.tasks' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tasks`')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tasks`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tasks`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\TaskController.php(19): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\TaskController->index()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TaskController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\ci4\\project1-ulfa\\public\\index.php')
CRITICAL - 2024-07-09 16:40:53 --> [Caused by] mysqli_sql_exception: Table 'ci4.tasks' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('SELECT *
FROM `tasks`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tasks`')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tasks`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tasks`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\TaskController.php(19): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\TaskController->index()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TaskController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\ci4\\project1-ulfa\\public\\index.php')
ERROR - 2024-07-09 16:47:52 --> mysqli_sql_exception: Table 'ci4.tasks' doesn't exist in C:\xampp\htdocs\ci4\project1-ulfa\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\MySQLi\Connection.php(310): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\ci4\project1-ulfa\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ci4\project1-ulfa\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ci4\project1-ulfa\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\ci4\project1-ulfa\app\Controllers\TaskController.php(19): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\ci4\project1-ulfa\system\CodeIgniter.php(933): App\Controllers\TaskController->index()
#8 C:\xampp\htdocs\ci4\project1-ulfa\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TaskController))
#9 C:\xampp\htdocs\ci4\project1-ulfa\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ci4\project1-ulfa\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ci4\project1-ulfa\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ci4\project1-ulfa\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\ci4\project1-ulfa\system\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-07-09 16:47:52 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'ci4.tasks' doesn't exist
[Method: GET, Route: task]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tasks`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\TaskController.php(19): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\TaskController->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TaskController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\ci4\\project1-ulfa\\public\\index.php')
CRITICAL - 2024-07-09 16:47:52 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'ci4.tasks' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tasks`')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tasks`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tasks`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\TaskController.php(19): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\TaskController->index()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TaskController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\ci4\\project1-ulfa\\public\\index.php')
CRITICAL - 2024-07-09 16:47:52 --> [Caused by] mysqli_sql_exception: Table 'ci4.tasks' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('SELECT *
FROM `tasks`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tasks`')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tasks`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tasks`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\TaskController.php(19): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\TaskController->index()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TaskController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\ci4\\project1-ulfa\\public\\index.php')
ERROR - 2024-07-09 16:54:25 --> mysqli_sql_exception: Table 'ci4.tasks' doesn't exist in C:\xampp\htdocs\project1-ulfa\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\project1-ulfa\system\Database\MySQLi\Connection.php(310): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\project1-ulfa\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\project1-ulfa\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\project1-ulfa\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\project1-ulfa\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\project1-ulfa\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\project1-ulfa\app\Controllers\TaskController.php(19): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\project1-ulfa\system\CodeIgniter.php(933): App\Controllers\TaskController->index()
#8 C:\xampp\htdocs\project1-ulfa\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TaskController))
#9 C:\xampp\htdocs\project1-ulfa\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\project1-ulfa\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\project1-ulfa\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\project1-ulfa\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\project1-ulfa\system\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-07-09 16:54:25 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'ci4.tasks' doesn't exist
[Method: GET, Route: task]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tasks`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\TaskController.php(19): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\TaskController->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TaskController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project1-ulfa\\public\\index.php')
CRITICAL - 2024-07-09 16:54:25 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'ci4.tasks' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tasks`')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tasks`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tasks`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\TaskController.php(19): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\TaskController->index()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TaskController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project1-ulfa\\public\\index.php')
CRITICAL - 2024-07-09 16:54:25 --> [Caused by] mysqli_sql_exception: Table 'ci4.tasks' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('SELECT *
FROM `tasks`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tasks`')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tasks`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tasks`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\TaskController.php(19): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\TaskController->index()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TaskController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project1-ulfa\\public\\index.php')
ERROR - 2024-07-09 17:01:22 --> mysqli_sql_exception: Table 'ci4.tasks' doesn't exist in C:\xampp\htdocs\project1-ulfa\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\project1-ulfa\system\Database\MySQLi\Connection.php(310): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\project1-ulfa\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\project1-ulfa\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\project1-ulfa\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\project1-ulfa\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\project1-ulfa\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\project1-ulfa\app\Controllers\TaskController.php(19): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\project1-ulfa\system\CodeIgniter.php(933): App\Controllers\TaskController->index()
#8 C:\xampp\htdocs\project1-ulfa\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TaskController))
#9 C:\xampp\htdocs\project1-ulfa\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\project1-ulfa\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\project1-ulfa\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\project1-ulfa\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\project1-ulfa\system\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-07-09 17:01:22 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'ci4.tasks' doesn't exist
[Method: GET, Route: task]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tasks`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\TaskController.php(19): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\TaskController->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TaskController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project1-ulfa\\public\\index.php')
CRITICAL - 2024-07-09 17:01:22 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'ci4.tasks' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tasks`')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tasks`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tasks`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\TaskController.php(19): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\TaskController->index()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TaskController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project1-ulfa\\public\\index.php')
CRITICAL - 2024-07-09 17:01:22 --> [Caused by] mysqli_sql_exception: Table 'ci4.tasks' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('SELECT *
FROM `tasks`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tasks`')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tasks`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tasks`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\TaskController.php(19): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\TaskController->index()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TaskController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('C:\\xampp\\htdocs\\project1-ulfa\\public\\index.php')
